README.md

```diff
@@ -38,3 +38,97 @@ List of `--preserve` options:
 - [CivicTheme UI Kit](https://github.com/civictheme/uikit)
 - [CivicTheme Drupal theme](https://github.com/civictheme/civictheme)
 - [Default content for CivicTheme](https://github.com/civictheme/civictheme_content)
+
+---
+
+# CivicTheme Setup Script for GovCMS
+
+This script automates the installation and configuration of CivicTheme, the `civictheme_govcms` support module, and a new custom subtheme within your GovCMS project.
+
+## Prerequisites
+
+1. **GovCMS Project:** You must have a GovCMS project already set up.
+2. **Ahoy:** `ahoy` must be installed and configured for your project (i.e., you can run `ahoy` commands).
+3. **Docker:** Docker must be running, as `ahoy` typically interacts with Docker containers.
+4. **`wget` and `tar`:** These command-line utilities must be available in your shell environment where you run the script.
+5. **Bash:** The script is written for Bash.
+6. **`curl`:** Required for downloading the script.
+
+## Installation
+
+1. **Download and Setup the Script:**
+   From your GovCMS project root directory, run:
+   ```bash
+   curl -o setup_civictheme.sh \
+     https://raw.githubusercontent.com/civictheme/civictheme_govcms/refs/heads/main/scripts/setup_civictheme.sh \
```

Comment:
@richardgaunt The curl command has been updated to point to the eventual correct path on the main branch. However, it won't work right now.

Use:
```
curl -o setup_civictheme.sh \
  https://raw.githubusercontent.com/IvanZugec/civictheme_govcms/refs/heads/CIVIC-2238-build-govcms-script/scripts/setup_civictheme.sh \
  && chmod +x setup_civictheme.sh
  ```

Reply:
Nice yes will change once we merge